schemas/taip-0/AddAgent.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://tap.rspv/schemas/AddAgent.json",
+  "title": "AddAgent",
```

Comment:
We might need a `RemoveAgent` (and maybe `UpdateAgent`). There was an `UpdateAgents` message in the original proposal but I'm concerned that it was overwriting the entire agents array (especially now that it contains policies).

@pelle how would we go about ensuring that agents don't overwrite other agents policies? e.g. either using `UpdateAgents` or `RemoveAgent` and then `AddAgent` with new policies set?